src/components/ProfilePage/LevelBox/ProgressionBar.tsx

```diff
@@ -56,31 +36,59 @@ export const ProgressionBar = ({ points, levelData }: ProgressionBarProps) => {
             />
             <ProgressionChartBg />
           </ProgressionChart>
-          <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
-            <CenteredBox>
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              marginTop: '24px',
+            }}
+          >
+            <Button
+              aria-label="Page Navigation"
+              variant="secondary"
+              size="medium"
+              styles={{
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                pointerEvents: 'none',
+                paddingLeft: '12px',
+                height: '32px',
+              }}
+            >
               <NoSelectTypography
```

Comment:
Duplicated with line 73, can you extract it to another component please?

Reply:
gud idea